src/components/infographics.tsx

```diff
@@ -1,188 +1,283 @@
+          </strong>
         </div>
-    );
+      </div>
+    </div>
+  );
 }
 
 // Economy
-export function EconomyGraphic(props: { indicators: Indicators, delay: number  }) {
-    const getTitle = () => props.indicators.medicalCosts > props.indicators.lockdownCosts ? 'Medical costs are high!' : 'Economy';
-    const calculateArea = (value: number): number => Math.sqrt(value * 100 / 3.14) * 2;
-    return (
-        <div 
-            className='m-2 w-full m-2 flex flex-col justify-center items-center text-white animate__animated animate__backInDown'
-            style={{ animationDelay: `${props.delay}s`}}
-        >
-            <Txt.Title value={getTitle()} col='white' />
-            <div className='flex flex-row'>
-                <div className='flex flex-col items-center'>
-                    <p className='p-2 text-lg font-medium'>Medical costs </p>
-                    <div className=''>
-                        <Txt.Text value={`$${props.indicators.medicalCosts} Billion`} col='white' />
-                    </div>
-                    <img
-                        src={medicalCoin}
-                        style={{ height: calculateArea(props.indicators.medicalCosts) }}
-                        alt='Medical costs'
-                        className=''
-                    />
-                </div>
-                <div className='mb-4 flex flex-col items-center'>
-                    <p className='p-2 text-lg font-medium'>Lost business activity</p>
-                    <div className=''>
-                        <Txt.Text value={`${props.indicators.lockdownCosts === 0 ? 'Very little or none' : `$${props.indicators.lockdownCosts}Billion`} `} col='white' />
-                    </div>
-                    <img
-                        src={lockdownCoin}
-                        style={{ height: calculateArea(props.indicators.lockdownCosts) }}
-                        alt='Lost business activity'
-                        className=''
-                    />
-                </div>
-            </div>
+export function EconomyGraphic(props: {
+  indicators: Indicators;
+  delay: number;
+}) {
+  const getTitle = () =>
+    props.indicators.medicalCosts > props.indicators.lockdownCosts
+      ? "Medical costs are high!"
+      : "Economy";
+  const calculateArea = (value: number): number =>
+    Math.sqrt((value * 100) / 3.14) * 2;
+  return (
+    <div
+      className="m-2 w-full m-2 flex flex-col justify-center items-center text-white animate__animated animate__backInDown"
+      style={{ animationDelay: `${props.delay}s` }}
+    >
+      <Txt.Title value={getTitle()} col="white" />
+      <div className="flex flex-row">
+        <div className="flex flex-col items-center">
+          <p className="p-2 text-lg font-medium">Medical costs </p>
+          <div className="">
+            <Txt.Text
+              value={`$${props.indicators.medicalCosts} Billion`}
+              col="white"
+            />
+          </div>
+          <Img
```

Comment:
Note the replacement of the `img` component - be sure to use this component to render images.